app/src/main/res/layout/home_activity.xml

```diff
@@ -1,7 +1,39 @@
-<?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.drawerlayout.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Shouldn't the first ``xmlns`` be on its own line? Please fix.

Reply:
We are showing the first xmlns with the tag on same line throughout app (codestyle)

Reply:
@BenHenning codeStyle followed in all xml.